src/utils/time.ts

```diff
@@ -3,22 +3,22 @@ import { FORMAT } from '../constants'
 /**
  * Adds a leading zero to a single-digit string and returns a two-digit string.
  * 
- * @param {String} s
- * @returns {String}
+ * @param {string} s
+ * @returns {string}
  */
-export const addLeadingZero = (s) => {
+export const addLeadingZero = (s: string) => {
   return `0${parseInt(s, 10)}`.slice(-2)
 }
 
```

Comment:
Every instance where `addLeadingZero()` is called is passing in a number when it expects a string